requirements.txt

```diff
@@ -1,5 +1,2 @@
-# This template is a low-dependency template. 
-# By default there is no requirements added here.
-# Add the requirements you need to this file.
-# or run `make init` to create this file automatically based on the template.
-# You can also run `make switch-to-poetry` to use the poetry package manager.
+SPARQLWrapper==2.0.1a0
```

Comment:
Be very careful when specifying that you are using an alpha release. These should not be used in a `reuirements.txt` or `setup.py` as they are really only released for developers for testing.